README.md

```diff
@@ -44,11 +44,11 @@ Example XML config:
 
 ### Configuration parameters
 
-- **UseHttp** : Determines whether logs are exported using gRPC or HTTP. Optional, by default false, meaning the gRPC exporter will be used.
-- **Endpoint** : Determines where the exporter should send logs. Used to set OtlpExporterOptions.Endpoint. 
-If left empty, the default values will be used (optional)
-- **Headers** : Used to set OtlpExporterOptions.Headers (optional)
-- **ServiceName** : Used to set the service.name resource (optional)
+- **Endpoint** : Determines where the exporter should send logs. Used to set OtlpExporterOptions.Endpoint. When not specified it will use environment-variable: `OTEL_EXPORTER_OTLP_ENDPOINT`
+- **UseHttp** : Determines whether logs are exported using gRPC or HTTP. (optional). When not specified it will use environment-variable: `OTEL_EXPORTER_OTLP_PROTOCOL`
```

Comment:
I'd formulate the second sentence as 'When not specified, the protocol will be determined by the environment-variable: `OTEL_EXPORTER_OTLP_PROTOCOL`'. Also I'd mention that it is false by default